content-repo/extra-docs/articles/setup-google-maps.md

```diff
@@ -0,0 +1,47 @@
+---
+id: google-maps
+title: Setup Google Maps in Cortex XSOAR using an Automation
+description: How to setup Google Maps in Cortex XSOAR using an automation.
+---
+
+You need to set up Google Maps in Cortex XSOAR before using either the [ShowOnMap](https://xsoar.pan.dev/docs/reference/scripts/show-on-map) or the `ShowLocationOnMap` automation. To use Google Maps, you need to create a `Google Maps Platform API` from a Google Cloud account that has [billing enabled](https://developers.google.com/maps/documentation/javascript/cloud-setup#billing). After creating the API, you need to add it to Cortex XSOAR. You can then use the `ShowOnMap` automation.
```

Comment:
Use consistent font for both the `ShowOnMap` and `ShowLocationOnMap` automations in the first sentence.

Reply:
You need to create a Google Cloud Platform project, not create an API. Then you need to enable the Maps JavaScript API and create an API key for the project.